django/applications/catmaid/control/stack.py

```diff
@@ -152,3 +152,15 @@ def stacks(request, project_id=None):
         'sort_keys': True,
         'indent': 4
     })
+
+@requires_user_role([UserRole.Annotate, UserRole.Browse])
+def stack_groups(request, project_id=None, stack_id=None):
+    stackgroup_links = StackStackGroup.objects \
+        .filter(stack=stack_id) \
+        .select_related('group_relation')
```

Comment:
Why is `group_relation` joined here? It seems below only the stack group ID is needed and we could basically use `.values_list('id', flat=True)` to get the result list of IDs.

Reply:
It's vestigial from including the relation type with the group links, before I decided to just retrieve the groups explicitly anyway (thinking I might put the raw/label enum on groups). Will remove.